atomic_reactor/plugins/pre_inject_yum_repo.py

```diff
@@ -83,6 +84,10 @@ def run(self):
 
         # Find out the USER inherited from the base image
         inspect = self.workflow.builder.base_image_inspect
-        inherited_user = inspect[INSPECT_CONFIG].get('User', '')
+        if self.workflow.builder.base_from_scratch:
```

Comment:
Any reason not to unconditionally do:
inherited_user = inspect.get(INSPECT_CONFIG, {}).get('User', '')
It matches the base_from_scratch case and won't affect the non-scratch case.

Reply:
it matches almost, I still want check that INSPECT_CONFIG exists for non-FROM-scratch base image